web_src/js/features/comp/ConfirmModal.js

```diff
@@ -0,0 +1,29 @@
+import $ from 'jquery';
+import {svg} from '../../svg.js';
+import {htmlEscape} from 'escape-goat';
+
+export async function confirmModal(opts = {content: '', buttonColor: 'green'}) {
+  return new Promise((resolve) => {
+    const i18n = window.config.i18n;
+    const $modal = $(`
+<div class="ui g-modal-confirm modal">
+  <div class="content">${htmlEscape(opts.content)}</div>
+  <div class="actions">
+    <button class="ui basic cancel button">${svg('octicon-x')} ${i18n.modal_cancel}</button>
```

Comment:
```suggestion
    <button class="ui basic cancel button">${svg('octicon-x')} ${htmlEscape(i18n.modal_cancel)}</button>
```

Reply:
No need, that's from our i18n.

We never do so in all other places.